games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/freegish/freegish.git#f53d915381c2dd9889dbebdff0541b04c6065067"
+PATCHES="freegish-1.0.patchset"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```

Comment:
Since you only tested x86_gcc2 build others should probably read as ?x86 and ?x86_64 as their status is unknown yet.